docs/content/guides/upgrade.md

```diff
@@ -21,7 +21,7 @@ Incompatibilities between previous and current versions may require specific ste
 - [Upgrade to 1.7.0 in OpenShift](#upgrade-to-170-in-openshift)
 
 {< important >}}
-When upgrading from <v1.7 to >=v1.7, any tracing settings that do not refer to an OpenTelemetry configuration will be erased. To deploy OpenTelemetry services and configure your mesh for OpenTelemetry tracing refer to the [Monitoring and Tracing]( {{< ref "/guides/monitoring-and-tracing.md" >}} ) guide.
+When upgrading from an NGINX Service Mesh prior to v1.7, any tracing settings that do not correspond to an OpenTelemetry configuration will be removed. To deploy OpenTelemetry services and configure your mesh for OpenTelemetry tracing refer to the [Monitoring and Tracing]( {{< ref "/guides/monitoring-and-tracing.md" >}} ) guide.
```

Comment:
Remove the `an` before NGINX Service Mesh. Part of the language we use is to avoid prefacing the product name with any articles.